build.gradle

```diff
@@ -16,6 +16,16 @@
 
 apply from: "gradle/dependencies.gradle"
 
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "com.diffplug.spotless:spotless-plugin-gradle:5.14.3"
```

Comment:
Should we track the spotless version in dependencies.gradle , alongside most other versions? I imagine it gets updated more frequently than most other dependencies we have.